tests/helpers/fixtures.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
I had to put a fixture into conftest.py because it has scope `session`, and it only work if imported once.
So I needed to put it there.
But yeah I can give it a try an move everything to conftest.py.

Reply:
Done